src/routes/index.tsx

```diff
@@ -88,7 +89,7 @@ const Routes = (): React.ReactElement => {
         }}
       />
 
-      <Route component={Home} exact path={HOME_ROUTE} />
+      <Route component={Home} exact path={[HOME_ROUTE, SAFE_ROUTES.DASHBOARD]} />
```

Comment:
This component will need to know whether it's on an 'addressed route'.

Do you think we should send a flag here or determine it within the component? Within the component we could just check for the existence of the param in the URL.

Reply:
Checking the URL sounds like a good idea! We will need it for some of the render logic i.e. to display the Add Safe/Load Safe Widget. I suggest we do that in a future PR once we have some of the Widgets ready.